packages/zipkin-instrumentation-axios/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "zipkin-js-instrumentation-axios",
```

Comment:
`zipkin-js-instrumentation-axios` -> `zipkin-instrumentation-axios`

Reply:
The name ` zipkin-instrumentation-axios` was taken up, any suggestion?

Reply:
Maybe `zipkin-instrumentation-axiosjs` or if we get @uschmann in the conversation maybe we can agree in a official one that could reuse the existing name?

Reply:
I already published this package with name: `zipkin-instrumentation-axiosjs` and openzipkin will be the maintainer